scripts/gulpfiles/release_notes.js

```diff
@@ -0,0 +1,201 @@
+      const pulls = pullRequestData.nodes;
+      cursor = pullRequestData.pageInfo.endCursor;
+    
+      for (let i = 0; i < pulls.length; i++) {
+        const pull = pulls[i];
+        const createdDate = Date.parse(pull.createdAt);
+        if (createdDate < goalDate) {
+          cursor = '';
+          break;
+        }
+
+        if (collaborators.has(pull.author.login)) {
+          continue;
+        }
+        let contributions = contributors.get(pull.author.login);
+        if (!contributions) {
+          contributions = [];
+          contributors.set(pull.author.login, contributions);
+        }
+        contributions.push({
+          title: pull.title,
+          url: pull.url,
+          number: pull.number
+        })
+      }
+
+    } catch (error) {
+        console.log(error);
+    }
+  } while (cursor);
+
+  let notes = '';
+  for (let [name, contributions] of contributors) {
+    notes += `* ${name}\n`;
+    for (let contribution of contributions) {
+      let {title, url, number} = contribution;
+      notes += `  * ([#${number}](${url})) ${title}\n`;
+    }
+  }
+  return notes;
+}
+
+async function getCollaborators(repo, headers) {
+  try {
+    const response = await fetch(baseUrl, {
+      method: "POST",
+      headers: headers,
+      body: makeCollaboratorQuery(repo),
+    });
+    const json = await response.json();
+    const collaborators =  json.data.repository.collaborators.nodes
+        .map((node) => node.login);
+    collaborators.push('dependabot');
+    return new Set(collaborators);
+  } catch (error) {
+    console.log(error);
+  }
+  return '';
+}
+
+async function getBreakingChanges(headers) {
+  return '';
+}
+
+async function getClosedIssues(headers) {
+  return '';
+}
+
+async function getMergedPrs(headers) {
+  return '';
+}
+
+function makeContributorQuery(repo, baseRef, cursor) {
+  return JSON.stringify({
+    query: `
+      query {
+        repository(name: "${repo}", owner: "google"){
+          pullRequests(first: 100, baseRefName: "${baseRef}",
+              ${cursor ? `after: "${cursor}"` : ''}
+              orderBy: {direction: DESC, field: CREATED_AT},
+              states: [OPEN, MERGED]) {
+            nodes{
+              author{
+                login
+              }
+              createdAt
+              title
+              url
+              number
+            }
+            pageInfo{
+              endCursor
+            }
+          }
+        }
+      }
+    `,
+  });
+} 
+
+function makeCollaboratorQuery(repo) {
+  return JSON.stringify({
+    query: `
+      query {
+        repository(name: "${repo}", owner: "google"){
+        collaborators(first:100){
+            nodes{
+              login
+            }
+          }
+        }
+      }
+    `,
+  });
+}
+
+function processContributors(goalDate, pullRequestData) {
+}
+
+const contributors = gulp.series(generateReleaseNotes);
+
+module.exports = {
+  contributors: contributors,
+}
```

Comment:
Are you expecting the `contributors` task to have other sub-tasks?  If not, just write:

```JS
module.exports = {
  contributors: generateReleaseNotes,
}
```